support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx

```diff
@@ -24,6 +24,7 @@ import { trackUserData, OPHAN_COMPONENT_ID_RETURN_TO_GUARDIAN } from '../utils/o
 import { trackComponentClick } from 'helpers/tracking/behaviour';
 import { getCampaignSettings } from 'helpers/campaigns';
 import type { CampaignSettings } from 'helpers/campaigns';
+import ContributionThankYouArticleShare from './ContributionThankYouArticleShare';
```

Comment:
tiny point - any chance we could move this up a couple of lines to be with all the other ContributionThankYouXXs?

Reply:
Done! ✅